src/views/FAQPage/FAQItem.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <Fragment>
+    <div class="faq-question" @click="() => toggleAccordion(question)">{{ question }}</div>
+    <div class="faq-answer">
+      <div class="answer">{{ answer }}</div>
+    </div>
+  </Fragment>
+</template>
+<script>
+import { Fragment } from 'vue-fragment';
+
+const FAQItem = {
+  name: 'faqitem',
+  components: {
+    Fragment,
+  },
+
+  props: ['question', 'answer', 'toggleAccordion'],
+};
+
+export default FAQItem;
+</script>
+
+<style scoped>
```

Comment:
we should favor rems over the use of ems. rems will be set in the main.css > html > font-size: 20px

Reply:
Okay, I just read little about them. To clarify, where `padding: 5px`, I need to use `padding: 0.25rem`. Is that correct?